src/reactviews/pages/ExecutionPlan/executionPlanGraph.tsx

```diff
@@ -154,11 +153,17 @@ export const ExecutionPlanGraph: React.FC<ExecutionPlanGraphProps> = ({
   		return percentage.toFixed(2) + '%';
 	};
 
+	const queryCostHeader = l10n.t({

```

Comment:
Can you add an example for this sort of function-based localization to the loc README file, too?

Reply:
I think it will be best if I link to the official documentation for the API. Since I don't want to keep updating the readme.